btrack/napari/main.py

```diff
@@ -252,11 +268,17 @@ def _run_tracker(
     """
     Runs BayesianTracker with given segmentation and configuration.
     """
-    with btrack.BayesianTracker() as tracker:
+    with btrack.BayesianTracker() as tracker, napari.utils.progress(
+        total=5
```

Comment:
Where does this `5` come from?

Reply:
I assume number of steps, in which case a variable?

Reply:
yeah exactly - it's the number of times we do `pbr.update(1)`